cobalt/__init__.py

```diff
@@ -1,4 +1,5 @@
-from .act import Act, AmendmentEvent, RepealEvent
+from .hierarchical import Act, AmendmentEvent, RepealEvent
 from .uri import FrbrUri
 
+# TODO: add other document types
```

Comment:
@longhotsummer what does this file do? should all the other document types be added?

Reply:
This means that you can do `from cobalt import Act`, as opposed to `from cobalt.act import Act`.